Sources/App/Controllers/SearchController.swift

```diff
@@ -7,7 +7,12 @@ struct SearchController {
 
     func show(req: Request) throws -> EventLoopFuture<HTML> {
         let query = req.query[String.self, at: "query"] ?? ""
-        return API.search(database: req.db, query: query)
+        let page = req.query[Int.self, at: "page"] ?? 1
+        let pageSize = req.query[Int.self, at: "page-size"] ?? Constants.searchPageSize
```

Comment:
Not sure about the parameter name `page-size` here, `pageSize`, `page_size`?

Reply:
I'd say maybe this doesn't need to be a query string parameter. The page number is certainly necessary, but page size can be fixed in the source. I think once we settle on a value, it won't need to change.